rts/lra/lra-test/src/main/java/io/narayana/lra/participant/api/StandardController.java

```diff
@@ -0,0 +1,85 @@
+/*
+ *******************************************************************************
+ * Copyright (c) 2018 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+package io.narayana.lra.participant.api;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.ws.rs.HeaderParam;
+import javax.ws.rs.NotFoundException;
+import javax.ws.rs.PUT;
+import javax.ws.rs.Path;
+import javax.ws.rs.WebApplicationException;
+import javax.ws.rs.client.ClientBuilder;
+import javax.ws.rs.client.Entity;
+import javax.ws.rs.client.WebTarget;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.UriInfo;
+import java.util.logging.Logger;
+
+import static io.narayana.lra.participant.api.ActivityController.ACTIVITIES_PATH;
+import static io.narayana.lra.participant.api.StandardController.ACTIVITIES_PATH3;
+import static org.eclipse.microprofile.lra.client.LRAClient.LRA_HTTP_HEADER;
+
+@ApplicationScoped
+@Path(ACTIVITIES_PATH3)
+public class StandardController {
+    public static final String ACTIVITIES_PATH3 = "activities3";
+    public static final String NON_TRANSACTIONAL_WORK = "work";
+
+    private static final Logger LOGGER = Logger.getLogger(ActivityController.class.getName());
+
+    @Context
+    private UriInfo context;
+
+    @PUT
+    @Path(NON_TRANSACTIONAL_WORK)
+    public Response work2(@HeaderParam(LRA_HTTP_HEADER) String lraId)
+            throws NotFoundException {
+
+        if (lraId != null) {
+            return Response.status(Response.Status.PRECONDITION_FAILED).entity(Entity.text("Unexpected LRA context")).build();
+        }
+
+        WebTarget resourcePath = ClientBuilder.newClient().target(context.getBaseUri()).path(ACTIVITIES_PATH).path("nestedActivity");
```

Comment:
I am assuming nestedActivity is already defined elsewhere in the project

Reply:
This is not a "request for changes" element of the review - just a query

Reply:
It is defined in the other resource/controller (ActivityController) that we are using for tests. But, it's a good point, I will add a static final for it (and for the other paths too).

Reply:
Agreed, I am going to turn all these @Path references to static final String